### PR TITLE
move validate checkbox to the first step. provide all validation resu…

### PR DESCRIPTION
**Pull request by IvanSheyenko**
…lts on the validation step.

### CODE REVIEW

**Comment by AndreyFilchenkov** [comment]
image

The validation details window contains uninformative 'success' messages. 
We should either:
* add more information about successful validation checks;
* or remove "success" messages from the 'validation results' window and leave just 'error' messages.

File: src/SIM.Tool.Windows/UserControls/Install/Validation/ValidationDetails.xaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿<Window x:Class="SIM.Tool.Windows.UserControls.Install.Validation.ValidationDetails"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:SIM.Tool.Windows.UserControls.Install.ParametersEditor"
+        mc:Ignorable="d"
+        Title="Advanced installation parameters" Height="450" Width="800" Loaded="Window_Loaded">
+    <Grid>
```
Comment (ViacheslavAfanasyev) [inline src/SIM.Tool.Windows/UserControls/Install/Validation/ValidationDetails.xaml:9]: Isn't clear to what validators/checks the successes are related: https://www.screencast.com/t/CpCakMORmgi4

The message string isn't wrapped:
https://www.screencast.com/t/wksrFwpQG

File: src/SIM.Sitecore9Installer/Validation/Validators/CertificateValidator.cs [hunk 2]
```diff
@@ -72,16 +72,11 @@ private X509Certificate2Collection FindInLocation(object searchValue, StoreLocat
         store.Open(OpenFlags.ReadOnly | OpenFlags.OpenExistingOnly);
         X509Certificate2Collection collection = store.Certificates;
 
-        fcollection = collection.Find(X509FindType.FindByIssuerName, searchValue, false);
+        fcollection = collection.Find(X509FindType.FindBySubjectName, searchValue, false);
```
Comment (ViacheslavAfanasyev) [inline src/SIM.Sitecore9Installer/Validation/Validators/CertificateValidator.cs:75]: Is it related to the re-implementation of a validation logic? Shouldn't it be in a separate PR?

File: src/SIM.Tool.Windows/UserControls/Install/Validation/ValidationDetails.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    {
+      ScrollViewer scrollviewer = sender as ScrollViewer;
+      if (e.Delta > 0)
+        scrollviewer.LineLeft();
+      else
+        scrollviewer.LineRight();
+      e.Handled = true;
+    }    
+
+    private void Btn_Close_Click(object sender, RoutedEventArgs e)
```
Comment (ViacheslavAfanasyev) [inline src/SIM.Tool.Windows/UserControls/Install/Validation/ValidationDetails.xaml.cs:44]: Can not continue the installation if the validation is canceled:
https://www.screencast.com/t/U9CyIW1MTkUC

Comment (DmitryKolinchuk) [reply src/SIM.Tool.Windows/UserControls/Install/Validation/ValidationDetails.xaml.cs:44]: I faced this issue while testing and registered a separate bug:
https://github.com/Sitecore/Sitecore-Instance-Manager/issues/426

Comment (AndreyFilchenkov) [reply src/SIM.Tool.Windows/UserControls/Install/Validation/ValidationDetails.xaml.cs:44]: Issue #426 is reproducible using the current code from the milestone/1.6.1 branch. So the bug is not caused by the code that is being added by this PR.

@ViacheslavAfanasyev  I propose to mark this particular change request as resolved one and consider the issue #426 as a separate bug.

File: src/SIM.Sitecore9Installer/Validation/Validators/CertificateValidator.cs (hunk 2 above)
Comment (IvanSheyenko) [reply src/SIM.Sitecore9Installer/Validation/Validators/CertificateValidator.cs:75]: it was interfering with testing of this change so I fixed it here.

File: src/SIM.Tool.Windows/UserControls/Install/Validation/ValidationDetails.xaml (hunk 1 above)
Comment (IvanSheyenko) [reply src/SIM.Tool.Windows/UserControls/Install/Validation/ValidationDetails.xaml:9]: Success messages are implemented separately.

Added some styles for word wrapping and better look.
